modules/webbrowser/include/modules/webbrowser/properties/propertycefsynchronizer.h

```diff
@@ -83,21 +83,29 @@ class IVW_MODULE_WEBBROWSER_API PropertyCefSynchronizer
 
     /**
      * Called due to cefQuery execution in message_router.html.
-     * Expects the request for be a JSON-format with key "id" of the property to change.
-     * Example {"id":"PropertyIdentifier", "value":"0.5"}
+     * Expects the request for be a JSON-format. See inviwoapi.js:
```

Comment:
"the request _for be_ a JSON-format"?

Do you mean something like this?
Expects the request to be a JSON data object